src/Upgrade/Application/Strategy/ReleaseApp/Processor/ModuleFetcher.php

```diff
@@ -74,22 +74,49 @@ protected function requirePackageCollection(PackageCollection $packageCollection
     {
         $requiredPackages = $this->packageCollectionMapper->getRequiredPackages($packageCollection);
         $requiredDevPackages = $this->packageCollectionMapper->getRequiredDevPackages($packageCollection);
+        $subPackages = $this->packageCollectionMapper->getUpdatedPackages($packageCollection);
 
         $response = $this->requirePackages($requiredPackages, static::REQUIRED_TYPE);
 
         if (!$response->isSuccessful()) {
             return $response;
         }
 
+        $responseSubPackages = $this->updateSubPackages($subPackages);
+
+        if (!$responseSubPackages->isSuccessful()) {
+            return $responseSubPackages;
+        }
+
         $responseDev = $this->requirePackages($requiredDevPackages, static::REQUIRED_DEV_TYPE);
 
         return new PackageManagerResponseDto(
             $responseDev->isSuccessful(),
-            implode(PHP_EOL, [$response->getOutputMessage(), $responseDev->getOutputMessage()]),
-            array_merge($response->getExecutedCommands(), $responseDev->getExecutedCommands()),
+            implode(PHP_EOL, [$response->getOutputMessage(), $responseSubPackages->getOutputMessage(), $responseDev->getOutputMessage()]),
+            array_merge($response->getExecutedCommands(), $responseDev->getExecutedCommands(), $responseSubPackages->getExecutedCommands()),
         );
     }
 
+    /**
+     * @param \Upgrade\Domain\Entity\Collection\PackageCollection $updatedSubPackages
+     *
+     * @return \Upgrade\Application\Dto\PackageManagerResponseDto
+     */
+    protected function updateSubPackages(PackageCollection $updatedSubPackages): PackageManagerResponseDto
+    {
+        if ($updatedSubPackages->isEmpty()) {
+            return new PackageManagerResponseDto(true, 'No new sub packages');
```

Comment:
It would be nice to have a more informative error.
F.e.: `There are no packages for the update.`